README.md

```diff
@@ -120,17 +120,17 @@ After configuring the build system:
 
 ## Using
 ### Querying
-The Search service can be queried for Products. See [Query Service API](https://github.com/connexta/ion-query-api) for information about paths and parameters
+The Search service can be queried for IRM. See [Query Service API](https://github.com/connexta/ion-query-api) for information about paths and parameters
```

Comment:
Food for thought. 
From a "RESTful" perspective, the Search service is querying for a resource. In this case the resource is "dataset". 
The most RESTful thing to do is to return a URI to the dataset itself. 

In a RESTful paradigm, IRM is an XML document that is one "representation" of a dataset. To get an IRM document, the client would send a GET to the dataset's URI with an accept header. The message would look like:

> GET /dataset
> HOST store.com
> ACCEPT: irm/xml

The media type is `irm/xml`. I believe that applications should come up with media types meaningful their applications. It makes a more complete model of the domain. 

Another alternative would be to use a query param,  but it is "RESTful"

`http://host/dataset?representation=irm`

Reply:
We cannot modify the API right now because it would break the thread. We have a ticket in our next sprint to re-write the Store API. I will copy your comment there.